### ISSUE TITLE
Google Assistant cannot play music

### ISSUE THREAD

**Issue by BaldEagleX02** [shivasiddharth/Assistants-Pi#94]
When I ask Google Assistant to play music it says: "I can't do that here, but you can ask me to do it on another device" or something similar.
How can I fix this? 
I have configured Spotify correctly.

**Comment by BaldEagleX02** [comment]
Yeah, you are right. I haven't thought it was considered a customization.

**Comment by BaldEagleX02** [comment]
Thank you for your help

**Comment by BaldEagleX02** [comment]
It doesn't work. If I say: "Play <my_playlist> from Spotify" it says: "Obtaining music links" and, after a few seconds, it reboots.

**Comment by BaldEagleX02** [comment]
Playing from YouTube works very well instead.

**Comment by shivasiddharth** [comment]
Have you input the spotify user-id in the config.yaml ?

**Comment by BaldEagleX02** [comment]
Yes

**Comment by BaldEagleX02** [comment]
I inserted Client ID, Client secret, my username and the Google API key for Youtube
